packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -47,75 +43,108 @@
   }
 
   .heading {
-    @apply text-n1-wrap
-    text-color-2
-    pointer-events-none
-    break-words
-    font-medium
-    duration-150
-    ease-in-out;
+    color: var(--calcite-color-text-2);
+    font-weight: 500;
+    overflow-wrap: break-word;
+    pointer-events: none;
+    transition-duration: 150ms;
+    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Can this be stored + referenced? It's used below as well. Do other components have a need for this value?

Reply:
Not sure if other components need it and I can't remember why this is even needed in Tile.  I can reference it at the least with a Sass variable.

Reply:
I removed these styles from `.heading` and `.description` since they don't seem to be needed because the host level rules cascade down to them, which is cleaner and less redundant.  I think it is out of scope of this PR to store and reference these values anyway since this is just about adding `scale` support, but in the forthcoming tile redesign PR I could look at adding a sass mixin for this.